sdk/servicebus/service-bus/samples/javascript/README.md

```diff
@@ -0,0 +1,86 @@
+
+3. Run whichever samples you like (note that some samples may require additional setup, see the table above):
+
+```bash
+node sendMessages.js
+```
+
+Alternatively, run a single sample with the correct environment variables set (step 3 is not required if you do this), for example (cross-platform):
+
+```bash
+npx cross-env SERVICE_BUS_ENDPOINT="<endpoint>" SERVICE_BUS_CONNECTION_STRING="<connection string>" QUEUE_NAME="<queue name>" node dist/basic.js
+```
+
+## Next Steps
+
+Take a look at our [API Documentation][apiref] for more information about the APIs that are available in the clients.
+
+[interactivelogin]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/interactiveLogin.js
+[scheduledmessages]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/scheduledMessages.js
+[receivemessagesstreaming]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/receiveMessagesStreaming.js
+[session]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/session.js
+[browsemessages]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/browseMessages.js
+[useproxy]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/useProxy.js
+[receivemessagesloop]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/receiveMessagesLoop.js
+[advanced-movingmessagestodlq]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/advanced/movingMessagesToDLQ.js
+[advanced-deferral]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/advanced/deferral.js
+[advanced-processmessagefromdlq]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/advanced/processMessageFromDLQ.js
+[advanced-sessionstate]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/advanced/sessionState.js
+[advanced-topicfilters]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/advanced/topicFilters.js
+[sendmessages]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/sendMessages.js
+[serviceprincipallogin]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/samples/javascript/servicePrincipalLogin.js
+[apiref]: https://docs.microsoft.com/javascript/api/@azure/service-bus
+[azsvcbus]: https://docs.microsoft.com/azure/service-bus-messaging/service-bus-create-namespace-portal
+[freesub]: https://azure.microsoft.com/free/
+[package]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/servicebus/service-bus/README.md
+
```

Comment:
Is this file going to be rendered on MS docs website?
I think we could include a note for running the browser based html samples at some place.
(Unless maybe if this is out of scope for now as such)

Reply:
Yes, these will show up on the docs.microsoft.com/samples browser. I would like to eventually handle the browser samples (some other packages have a browser sample), but there's no structure for that yet. Either we need to include them in the packages with all the Node samples and provide ts/js versions of both, or we need to have a separate way to get to the browser samples.

I still need to edit the service-bus/samples/README.md file to match these changes, and that's one place that I could mention that.